mgtool/configuration.go

```diff
@@ -40,7 +40,7 @@ func GetGitRootPath(path string) string {
 	if err := c.Run(); err != nil {
 		panic(err)
 	}
-	return filepath.Join(output.String(), path)
+	return filepath.Join(strings.TrimSuffix(output.String(), "\n"), path)
```

Comment:
I typically use strings.TrimSpace for even more fool-proof whitespace trimming

Reply:
Ah ofc, that does the job. Also easier to read :) Updated